Counterstrike Soviet mission Top o' The World ported

AngryBirdz opened the pull request:
I've just finished porting this mission. Tell me if you spot something wrong ;)

Updated: I've fixed the rebase issues.

pchote commented:
Looks like some rebase issues have crept into the commits here. You'll need to edit the files to remove the `<<<<<<< HEAD` through `>>>>>>> top-o-the-world` lines, and pick the right version of the code in between.

Commit:
Counterstrike Soviet mission Top o' The World ported

I've just finished porting this mission. Tell me if you spot something wrong ;)

Update: Fixed a few things in the lua and rules.

AngryBirdz commented:
Fixed

dragunoff commented on mods/ra/maps/top-o-the-world/scu36ea.lua:
```diff
@@ -0,0 +1,371 @@
+	if RemainingTime == DateTime.Minutes(30) then
+		Media.PlaySpeechNotification(player, "ThirtyMinutesRemaining")
+	elseif RemainingTime == DateTime.Minutes(20) then
+		Media.PlaySpeechNotification(player, "TwentyMinutesRemainingg")
```
There's a typo - this should be `"TwentyMinutesRemaining"`

pchote commented:
Can we get some testing and reviewing here?

abcdefg30 reviewed (commented):
Looks good to me otherwise.

obrakmann commented on mods/ra/maps/top-o-the-world/scu36ea.lua:
```diff
@@ -0,0 +1,361 @@
+
+Tick = function()
+
+-- Timer Setup
```
This would have been the perfect use case for my time limit PR. Oh well.

pchote replied:
Do you want to update this in your PR?

obrakmann replied:
I could rebase it now and do it, yes.

obrakmann reviewed (approved):
Mission looks good, :+1:

obrakmann commented:
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/86b756a140b85d134aa3619a3a08db6d49f88b12)
